Delicate-Butterflies/Bangazon-Site
# Mb order history

Related Issue: #17 #16 

Description:
User can see account details when user clicks account on nav bar
From Account view, user can go to edit view and edit account details, return to account
From Account view, user can go to order history view and view past orders
From Order History view, user can view individual order details (products and total price)
From Account view, user can go to payment types view
From Payment type view, user can go to add-payment view 

PR Testing Steps:
 Go to account:
- view account details
- edit account details
- view orders and order details
- make sure payment type paths work

Check list before you make a PR:
